activities/tool-management/g-suite.md

```diff
@@ -0,0 +1,23 @@
+---
+type: resource
+---
+
+# G Suite
+
+G Suite is a suite of cloud computing, productivity and collaboration tools, software and products developed by Google.
+We use G Suite for email, calendaring and internal document management.
+
+## Using G Suite
+
+Staff are issued a G Suite account when they join.
+Google publishes [extensive support documentation](https://support.google.com/).
+
+## G Suite management
+
+G Suite's (user) administration is managed by the Operations coordinator. The Operations coordinator as well as the Chief executive a have full adminstrative privileges. The Operations coordinator is responsible for user management. She ensures proper on-boarding and off-boarding policies are in place and also creates and manages groups, aliases, domain management and 3rd party app integrations.
```

Comment:
remove "a" after "executive" and before "have" + please add (if deemed a good add-on) in sentence "off-boarding policies are in place + and enforced" to make it even more complete, as writing / having policies is one thing, enforcing it is something else.

Reply:
Done!